themes/harbor/source/scss/harbor/module/_module--cncf.scss

```diff
@@ -0,0 +1,83 @@
+.module--cncf{
+
+  $item-v-gap:30px;
+
+  @mixin item-title-gap($count){
+    @for $i from 1 to ($count+1){
+      &:nth-last-of-type(#{$i}){
+        padding-top:$item-v-gap!important;
+      }
+    }
+  }
+  @mixin flex-child-item-grid(){
+
+    $num-of-items:(
+            "mobile":2,
+            "tablet":2,
+            "desktop":4
+    );
+
+    flex-basis: calc(100%/#{map_get($num-of-items,mobile)});
+    padding:$item-v-gap $item-gutter $item-gutter $item-gutter;
+    box-sizing: border-box;
+    //@include item-title-gap(map_get($num-of-items,mobile));
+    @include breakpoint(tablet){
+      flex-basis: calc(100%/#{map_get($num-of-items,tablet)});
+      padding:$item-v-gap $item-gutter $item-gutter $item-gutter;
+      box-sizing: border-box;
+      //@include item-title-gap(map_get($num-of-items,tablet));
+    }
+    @include breakpoint(desktop){
+      flex-basis: calc(100%/#{map_get($num-of-items,desktop)});
+      padding:$item-v-gap $item-gutter $item-gutter $item-gutter;
+      box-sizing: border-box;
+      //@include item-title-gap(map_get($num-of-items,desktop))
+    }
+  }
+
+  $item-gutter:10px;
+
+  //background: $harbor__faded_yellow;
+
+  //@include curve(full-curve-reverse);
+  padding-top:30px;
```

Comment:
Setting this to `0px` helps remove some of that white space. I think that's all that's needed.